install_base_ubuntu.sh

```diff
@@ -42,6 +42,16 @@ install_sdk() {
     fi
 }
 
+install_nodejs() {
+    # NodeJS v8+ is required, Ubuntu 16.04 LTS supports only an older version.
+    # As a special case we can install it as a snap package
+    if grep 16.04 /etc/lsb-release >/dev/null; then
+        sudo snap install node --classic --channel=8
+    elif grep Ubuntu /etc/lsb-release >/dev/null; then
```

Comment:
Actually I'm realizing that
```
     elif grep Ubuntu /etc/lsb-release >/dev/null; then 
```
could just be `else`, since that script is only expected to be run on Ubuntu-ish distributions.